src/mn_setup.c

```diff
@@ -3564,6 +3564,11 @@ boolean MN_SetupResponder(menu_action_t action, int ch)
         }
     }
 
+    if (highlight_item != set_item_on)
+    {
+        current_menu[set_item_on].m_flags |= S_HILITE;
+    }
```

Comment:
But in the same function we have this code:
https://github.com/fabiangreffrath/woof/blob/d8eac1c38ba1b53bf66a7838f14bbbcf4e1416e0/src/mn_setup.c#L3446-L3449

To be honest, I don't remember the highlight_item logic. I will look into it later.

Reply:
Yes, that part removes the highlighting from the `highlight_item` item, whereas my added code sets it for the `highlight_item` item.

Reply:
I thought that maybe this should be fixed in the `MN_ResetMouseCursor` function. I'll check it out.

Reply:
> I'll check it out.

Okay!